_posts/2024-01-29-advanced-authorization.adoc

```diff
@@ -0,0 +1,95 @@
+---
+title: Advanced Authorization
+layout: post
+author: Ilya Buziuk
+description: >-
+  The article describes how to configure advanced authorization in Eclipse Che
+categories: []
+keywords: ['postgresql']
+slug: /@ilya.buziuk/advanced-authorization
+---
+
+Even though the current trend is instead of having a gigantic monolith {kubernetes} cluster split the infrastructure into a number of 'Fit For Purpose' clusters, administrators still want to provide granular access and restrict certain functionality for particular users.
```

Comment:
```suggestion
Even though the current trend is to split an infrastructure into a number of 'Fit For Purpose' clusters instead of having a gigantic monolith {kubernetes} cluster, administrators still want to provide granular access and restrict certain functionality for particular users.
```